### ISSUE TITLE
Symlink isn't updated when updating the prefix_dir attribute

### ISSUE THREAD

**Issue by nicka** [Dynatrace/Dynatrace-Chef#11]
For example in production we use an external AWS EBS volume:
```
"dynatrace": {
    "server": {
        "linux": {
            "installer": {
                "prefix_dir": "/persistent/opt"
            }
        },
    },
}
```

**Comment by metmajer** [comment]
Hi @nicka. I will think about a solution for this, soon.

**Comment by pozieblo** [comment]
I tested the following secnearios on a recent version of the cookbook:

"dynatrace": {
    "server": {
        "linux": {
            "installer": {
                "prefix_dir": "/opt/other"
            }
        },
    },
}
and launched the server recipe. I then reinstalled the server with another prefix (“/opt/other2”) without any issue. Finally I took an old version (commit f6e6a4595d from Sept 23, 2015) trying to reproduce the issue without success – the symlink was created and the server worked fine. Closing as I am unable to reproduce it.
